src/test/java/io/projectriff/samples/shopping/inventoryapi/article/ArticleRepositoryTest.java

```diff
@@ -50,6 +50,32 @@ void finds_all_articles() {
   }
 
   @Test
+  @DisplayName("Finds articles by SKU")
+  void find_article_by_sku() {
+    jdbcHelper.save(article);
+    jdbcHelper.save(otherArticle);
+
+    Article found = repository.findBySku(article.getSku());
+
+    assertThat(repository.findAll()).containsExactly(article, otherArticle);
+    assertThat(found.getSku()).isEqualTo(article.getSku());
+  }
+
+  @Test
+  @DisplayName("Updates quantity for articles by SKU")
+  void update_article_by_sku() {
+    jdbcHelper.save(article);
+    jdbcHelper.save(otherArticle);
+
+    int count = repository.updateQuantityBySku(article.getSku(), 12, 11);
+
+    Article updatedArticle = new Article("some SKU", "some name", "description", BigDecimal.TEN, null, 11);
```

Comment:
the constructor argument should be calls to the getters of `article` (except for the quantity), shouldn't they?

Reply:
I though about it but that became a very long statement - I can change that though